### PR TITLE
Full example for framework

### PR DESCRIPTION
**Pull request by g-krisztian**
Created a new PR for restart conversation.

### CODE REVIEW

File: examples/acl/src/backend/app/models/posts.clj [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+(defn delete-query
+  [{{{id :id} :query-params} :request
+    :as                      state}]
+  (xiana/ok (assoc state :query (cond-> (delete-from :posts)
+                                  id (where [:= :id (UUID/fromString id)])))))
+
+(defn fetch-by-ids-query
+  [{{{ids :ids} :body-params} :request
+    :as                       state}]
+  (xiana/ok (assoc state :query (cond-> (-> (select :*)
```
Comment (ianffcs) [inline examples/acl/src/backend/app/models/posts.clj:40]: maybe something like this below?
 ```
 (defn fetch-by-ids-query
  [{{{ids :ids} :body-params} :request
    :as                       state}]
   (xiana/ok (assoc state :query (cond-> (-> (select :*)
                                            (from :posts))
                                  (coll? ids) (where [:in :id (map #(UUID/fromString %) ids)])
                                  :else       (where [:in :id (map #(UUID/fromString %) [ids])])))))
```

File: examples/acl/src/backend/app/models/users.clj [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                          :password
+                          :last_login
+                          :is_superuser
+                          :username
+                          :first_name
+                          :last_name
+                          :email
+                          :is_staff
+                          :is_active])]
+    (cond-> u
```
Comment (ianffcs) [inline examples/acl/src/backend/app/models/users.clj:22]: Do you want this to run a single time?

Comment (g-krisztian) [reply examples/acl/src/backend/app/models/users.clj:22]: > Do you want this to run a single time?

I don't like it too, looks like muuntaja doesn't converts booleans from json body. Needs more inspection.

File: src/framework/components/interceptors/wrap.clj [hunk 3]
```diff
@@ -0,0 +1,24 @@
+(ns framework.components.interceptors.wrap
+  (:require
+    [xiana.core :as xiana]))
+
+(defn interceptor
```
Comment (ianffcs) [inline src/framework/components/interceptors/wrap.clj:5]:
```suggestion
(defn interceptor
  [{:keys [enter leave error]}]
  (cond-> {}
    enter (assoc :enter (fn [state] (xiana/ok (enter state))))
    leave (assoc :leave (fn [state] (xiana/ok (leave state))))
    error (assoc :error (fn [state] (xiana/error (error state))))))
```

**Review by ianffcs — APPROVED** [review]

**Review by narocath — APPROVED** [review]

**Pull request merged**